config-mobi.toml

```diff
@@ -102,4 +102,4 @@ url = "https://twitter.com/puzzleitc"
 [[params.links.user]]
 name = "LinkedIn"
 icon = "fab fa-linkedin-in"
-url = "https://linkedin.com/company/puzzle-itc"
\ No newline at end of file
+url = "https://linkedin.com/company/puzzle-itc"
```

Comment:
Wollen wir das gleich auf acend ändern?

Reply:
Ja gute Idee, aber besser in einem separaten PR.